### PR TITLE
Feature/1857 feature card block

### PR DESCRIPTION
**Pull request by reesedownes**
### Related Ticket(s)

https://app.zenhub.com/workspaces/ibmcom-library-5d449f3642eb1962336cbe52/issues/carbon-design-system/ibm-dotcom-library/1857

### Description

Develop new FeatureCardBlockLarge block using functional and design specs

- FeatureCardBlockLarge will be added to menu of patterns

### CODE REVIEW

File: packages/react/src/patterns/blocks/FeatureCardBlockLarge/FeatureCardBlockLarge.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+const FeatureCardBlockLarge = ({ card, onClick }) => {
+  const hasCopy = classNames({
+    [`${prefix}--feature-card-block-large_no-copy-text`]: !card.copy,
+  });
+  return (
+    card.eyebrow &&
+    card.heading &&
+    card.cta && (
+      <section
+        className={`${prefix}--feature-card-block-large ${hasCopy}`}
```
Comment (jeffchew) [inline packages/react/src/patterns/blocks/FeatureCardBlockLarge/FeatureCardBlockLarge.js:35]: I'm wondering if you can truncate this a bit and lose the `hasCopy`:
```
className={classNames(
  `${prefix}--feature-card-block-large,
  [`${prefix}--feature-card-block-large_no-copy-text`]: !card.copy
)}
``

File: packages/react/src/patterns/blocks/FeatureCardBlockLarge/__stories__/FeatureCardBlockLarge.stories.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+storiesOf('Patterns (Blocks)|FeatureCardBlockLarge', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      sidebar: readme,
+    },
+  })
+  .add('Default', () => {
+    const cardheading = text(
+      'Card heading(required):',
```
Comment (jeffchew) [inline packages/react/src/patterns/blocks/FeatureCardBlockLarge/__stories__/FeatureCardBlockLarge.stories.js:24]: We're starting to add the actual prop name in parenthesis next to the description, can you update to something like : 
```
const cardheading = text('Card heading(required) (heading):', 'Explore AI use cases in all industries');
```

```

File: packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+          @include carbon--type-style(expressive-heading-04, true);
+
+          margin-bottom: rem(28px);
+        }
+
+        .#{$prefix}--card__copy {
+          p {
+            font-size: 0.875rem;
+            font-weight: 400;
+            line-height: 1.2rem;
```
Comment (jeffchew) [inline packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss:109]: This is pretty hard coded, is there a token or font we can use here instead?

cc: @kennylam @annawen1

File: packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+        overflow-y: hidden;
+      }
+
+      &__wrapper {
+        background: $inverse-02;
+        min-height: 50%;
+        height: auto;
+
+        @include carbon--breakpoint('xlg') {
+          padding: rem(32px);
```
Comment (jeffchew) [inline packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss:82]: @reesedownes are there spacing tokens for these?

File: packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+        }
+
+        @include carbon--breakpoint('max') {
+          padding: rem(64px);
+        }
+
+        .#{$prefix}--card__eyebrow {
+          margin-bottom: rem(14px);
+          color: #ffffff;
```
Comment (jeffchew) [inline packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss:91]: There should be a token for this one too...

**Comment by annawen1** [comment]
Also need to export it in `.../packages/react/src/patterns/blocks/index.js`

File: packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+    }
+    @include carbon--breakpoint('xlg') {
+      padding-top: aspect-ratio(2, 1);
+    }
+
+    &__card.#{$prefix}--card--inverse .#{$prefix}--card__cta,
+    &__card.#{$prefix}--card--inverse .#{$prefix}--card__cta:visited {
+      float: none;
+      color: $inverse-link;
+      font-size: 2rem;
```
Comment (jeffchew) [inline packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss:36]: We shouldn't be hard coding font sizes here...

File: packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss (hunk 3 above)
Comment (annawen1) [reply packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss:109]: I think the type token is specified in the visual specs

Comment (annawen1) [reply packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss:109]: I believe `copy` is using `body-long-02`

File: packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+
+        @include carbon--breakpoint('max') {
+          padding: rem(64px);
+        }
+
+        .#{$prefix}--card__eyebrow {
+          margin-bottom: rem(14px);
+          color: #ffffff;
+
+          @include carbon--breakpoint-between(1px, 'sm') {
```
Comment (annawen1) [inline packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss:93]: is this media query needed here?

File: packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss (hunk 5 above)
Comment (annawen1) [reply packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss:91]: Looks like eyebrow uses

`color: $inverse-01`

File: packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss (hunk 4 above)
Comment (annawen1) [reply packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss:82]: @reesedownes you can use carbon tokens defined here: https://www.carbondesignsystem.com/guidelines/spacing

I tend to use the `carbon--spacing-` tokens more for spacing within a component and the `carbon--layout-` ones for spacing between components

Comment (reesedownes) [reply packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss:82]: where do i find the spacing tokens?

File: packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss (hunk 5 above)
Comment (reesedownes) [reply packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss:91]: the color i've fixed, so i want to check if it's for the margin-bottom

File: packages/styles/scss/patterns/blocks/feature-card-block-large/README.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
+### SCSS
+
+#### Usage
+
+Import the package css into the top of your main CSS file.
+
+```css
+@import '@carbon/ibmdotcom-styles/scss/patterns/blocks/feature-card/feature-card';
```
Comment (annawen1) [inline packages/styles/scss/patterns/blocks/feature-card-block-large/README.md:8]: should be feature-card-block-large right?

Comment (annawen1) [reply packages/styles/scss/patterns/blocks/feature-card-block-large/README.md:8]: Think this README needs to be updated

**Pull request merged**